CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ This page provides information about contributing code to the Jenkins core codeb
      - In Jenkins project we usually use [OpenJDK],
   but you can use other JDKs as well.
      - Java 9+ is **not supported** in Jenkins.
```

Comment:
While we're at it… @oleg-nenashev Time to clean this up?

Reply:
It is